### PR TITLE
WIP: feat(InteractionsTab): add the Interactions display (ONLY UI)

### PR DESCRIPTION
**Pull request by RoeiRom**

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts [hunk 1]
```diff
@@ -43,11 +44,17 @@ const useInteractionsTab = (input: useInteractionsTabInput) :  useInteractionsTa
         // TODO: delete the event from db
     }
 
+    const loadInteractions = () => {
+        // TODO: add loading from DB
+        axios.get('/intersections?epidemioligyNumber=111').then(res => console.log(res));
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts:49]: console .log 🤷‍♀️

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx [hunk 2]
```diff
@@ -1,23 +1,94 @@
...
+                    name: 'עומר שמיר', 
+                    phoneNumber: '058-5161606'
+                }
+            ]
+        }
+    ]);
 
-    const { getDatesToInvestigate, createNewInteractionEvent, cancleNewInteractionEvent, confirmNewInteractionEvent } = 
+    const interactionsPerDate = React.useMemo<Map<number, Interaction[]>>(() => {
+        const returnedMap = new Map<number, Interaction[]>();
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx:67]: significant variable names would be nice 😄

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts (hunk 1 above)
Comment (RoeiRom) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts:49]: It is a stupid axios request to get information from the be

Comment (RoeiRom) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts:49]: it is a stub that will be replaced

**Review by sandymash — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/InteractionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { Card, Collapse, IconButton, Typography, Grid, Divider } from '@material-ui/core';
+import { KeyboardArrowDown, KeyboardArrowLeft, Edit, Delete } from '@material-ui/icons';
+
+import Interaction from 'models/Interaction';
+
+import useStyle from './InteractionCardStyles';
+
+const InteractionCard: React.FC<Props> = (props: Props) => {
```
Comment (daniel7764) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/InteractionCard.tsx:10]: Me and Maya are calling what you are displaying here an 'InteractionEvent' so i'd change the name to 'InteractionEventCard'

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/InteractionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                        <Edit />
+                    </IconButton>
+                    <IconButton>
+                        <Delete />
+                    </IconButton>
+                </div>
+            </div>
+            <Collapse in={areDetailsOpen}>
+                <Grid container className={classes.gridContainer}>
+                    {/* place name row */}
```
Comment (roymaniv) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/InteractionCard.tsx:42]: למחוק הערות בכל הקובץ

**Review by daniel7764 — APPROVED** [review]

**Review by roymaniv — DISMISSED** [review]

**Review by daniel7764 — DISMISSED** [review]

**Review by daniel7764 — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Pull request merged**